### PR TITLE
Comm-counts perf testing

### PR DESCRIPTION
**Pull request by awallace-cray**
First cut at Jenkins job script for Comm-counts perf testing, req by @mppf . 
Also see https://github.com/chapel-lang/chapel/pull/6604

Perf data goes into /data/sea/chapel/NightlyPerformance/chapcs/cc
(http://chapel.sourceforge.net/perf/chapcs/cc/)

### CODE REVIEW

File: util/cron/test-perf.chapcs.cc.bash [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.chapcs.cc"
+
+SHORT_NAME=cc
+START_DATE=08/17/17
+
+perf_args="-performance-description $SHORT_NAME -sync-dir-suffix $SHORT_NAME -perflabel cc-"
+perf_args="${perf_args} -performance -numtrials 5 -startdate $START_DATE"
+
+# make test will fail on chapcs because Gasnet wants to use ibv, and that warning
```
Comment (awallace-cray) [inline util/cron/test-perf.chapcs.cc.bash]: That should read "make check" will fail..
Will correct that for final.

File: util/cron/test-perf.chapcs.cc.bash [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Run -perflabel cc- performance tests on a chapcs machine with gasnet
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='chapcs'
```
Comment (mppf) [inline util/cron/test-perf.chapcs.cc.bash]: I don't really know the impact of these variables but I can't tell if this should be chapcs-comm-counts, say

File: util/cron/test-perf.chapcs.cc.bash [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+source $CWD/common-perf.bash
+source $CWD/common-gasnet.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.chapcs.cc"
```
Comment (mppf) [inline util/cron/test-perf.chapcs.cc.bash]: Can we call it perf.chapcs.comm-counts or something like that?

File: util/cron/test-perf.chapcs.cc.bash [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+SHORT_NAME=cc
+START_DATE=08/17/17
+
+perf_args="-performance-description $SHORT_NAME -sync-dir-suffix $SHORT_NAME -perflabel cc-"
```
Comment (mppf) [inline util/cron/test-perf.chapcs.cc.bash]: What does performance-description do? 'cc' might be too short if it's for something human-readable.

**Comment by mppf** [comment]
This seems good to me, but I'm not sure about the various names yet.

**Comment by awallace-cray** [comment]
I'm not sure about the various names, either!  I regard this as an initial guess, which should at least do no harm- that was why I added that new "cc" subdir. 

I know about some patterns and constraints on the job name.  For example, instead of job name "chapcs-perf-test-perf.chapcs.cc" (with script file test-perf.chapcs.cc.bash), it could easily have job name "performance-test-perf.chapcs.comm-count" (test-perf.chapcs.comm-count.bash).  

I know almost nothing about the perf data related names. Worse, this looks like a new type of perf test, not just another instance of some existing pattern.  

IMO, we could at least get these builds going and put some data out there, then consult with Elliot when he gets back.

**Comment by mppf** [comment]
@awallace-cray - sure that is fine with me. thanks.

**Pull request merged**